src/Plugin/Validation/Constraint/ValidOgRoleConstraintValidator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\og\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+/**
+ * Checks if referenced OG role is valid.
+ */
+class ValidOgRoleConstraintValidator extends ConstraintValidator {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate($value, Constraint $constraint) {
+    /* @var \Drupal\Core\Field\FieldItemInterface $value */
```

Comment:
Inline @var declarations should use the /** */ delimiters

```suggestion
    /** @var \Drupal\Core\Field\FieldItemInterface $value */
```